examples/redis/Examples.StackExchangeRedis/README.md

```diff
@@ -0,0 +1,5 @@
+# StackExchangeRedis Instrumentation for OpenTelemetry .NET - Example
+
+An example application that shows how to use
+`OpenTelemetry.Instrumentation.StackExchangeRedis` to capture traces of outgoing
+calls to Redis.
```

Comment:
you can maybe include a example output when the program is ran, with the pre-req (redis)